app/assets/javascripts/helpers/protocol_handler.es6

```diff
@@ -0,0 +1,21 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+Diaspora.ProtocolHandler = class {
```

Comment:
This file needs to be a normal `.js` file, because as far as I know the es6 stuff is only in `develop`.

Reply:
... tragic.